src/style.scss

```diff
@@ -98,47 +114,54 @@ body {
       }
     }
     div.details {
-      span.netmask {
-        display: inline-block;
-        width: 15%;
-      }
-      span.base {
-        display: inline-block;
-        width: 15%;
-      }
-      span.broadcast {
-        display: inline-block;
-        width: 15%;
+      display: inline-flex;
+      justify-content: space-around;
+      width: 100%;
+      @media (min-width: 640px) {
+        width: 640px;
       }
-      span.count {
-        display: inline-block;
-        width: 15%;
+      span {
+        display: block;
       }
-      span.first {
-        display: inline-block;
-        width: 15%;
+    }
```

Comment:
Break is just a dummy flex element full width and no height to force a row break.